a2ml/server/config.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+class Config:
+    def __init__(self):
+        self.celery_broker_url = os.environ.get('CELERY_BROKER_URL', 'redis://localhost:6379/0')
+        self.debug = os.environ.get('DEBUG', 'false').lower() == 'true'
+        self.redis_host = os.environ.get('REDIS_HOST', 'localhost')
+        self.redis_port = os.environ.get('REDIS_PORT', 6379)
```

Comment:
Do we need redis-specific config? The broker url should handle it so we don't need `REDIS_HOST` and `REDIS_PORT`

Reply:
it's for WebSocket notificator